ui/js/redux/selectors/claims.js

```diff
@@ -171,7 +171,8 @@ export const selectMyClaimsWithoutChannels = createSelector(
 
 export const selectAllMyClaimsByOutpoint = createSelector(
   selectMyClaimsRaw,
-  claims => new Set(claims.map(claim => `${claim.txid}:${claim.nout}`))
+  claims =>
+    new Set(claims && claims.map(claim => `${claim.txid}:${claim.nout}`))
```

Comment:
Won't this be `new Set(false)` when claims is false? This may work but is probably slightly wrong.